### DISCUSSION TITLE
SFT的train epoch设置多少

### DISCUSSION THREAD

**Discussion by lychee-2724540853** [ymcui/Chinese-LLaMA-Alpaca-2#159] [Q&A]
默认是1，那这个应该怎么设置呢？如果和数据量相关，有什么参照的标准吗？比方说20w跳数据的话，epoch设置多少合适？

**Comment by johnnyhuangcs** [comment]
我个人从实验中感觉epoch的大小得看loss，没有特定的数值，只要loss还在下降就得增大epoch数，当然也取决于你的计算资源大小
